packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -11,7 +10,7 @@ import {
 } from 'components/form-fields'
 import { moodMap } from 'utils/Moods'
 
-import { getTrackFieldName } from '../forms/utils'
+import { getTrackFieldName } from '../hooks'
```

Comment:
I know I did this, but feels weird to import a regular function from a hooks file 🤔